packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -17546,7 +17546,8 @@ void main() {
     await tester.pumpWidget(widget);
     await tester.showKeyboard(find.byType(EditableText));
     await tester.pumpAndSettle();
-    expect(scrollController.offset, 75.0);
+    // Regression test for https://github.com/flutter/flutter/issues/154615
+    expect(scrollController.offset, 0.0);
```

Comment:
Testing this out in an app, I don't think this is correct to change.
This test is verifying that when the keyboard is presented, with focus on the text field, it is then scrolled into view. An offset of zero means it did not scroll at all.

Reply:
See https://github.com/flutter/flutter/pull/141671 for more context

Reply:
Ah I see, I'll probably have to improve my code. It really works for specific cases.I will work better on this issue.